Source/CesiumRuntime/Private/Tests/CesiumLoadTest.cpp

```diff
@@ -299,15 +298,93 @@ bool RunLoadTest(const size_t locationIndex) {
         loadElapsedTime);
   }
 
+  if (afterTest) {
+    afterTest(context);
+  }
+
   return !timedOut;
 }
 
 bool FCesiumLoadTestDenver::RunTest(const FString& Parameters) {
-  return RunLoadTest(1);
+  return RunLoadTest(setupForDenver);
 }
 
 bool FCesiumLoadTestGoogleplex::RunTest(const FString& Parameters) {
-  return RunLoadTest(0);
+  return RunLoadTest(setupForGoogleTiles);
+}
+
+bool FCesiumLoadTestMontrealPointCloud::RunTest(const FString& Parameters) {
+  auto setup = [this](LoadTestContext& context) {
+    FVector targetOrigin(-73.616526, 45.57335, 95.048859);
+    FString ionToken(
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI2NmZhZTk4NS01MDFmLTRjODgtOTlkYy04NjIwODhiZWExOGYiLCJpZCI6MjU5LCJpYXQiOjE2ODg1MTI4ODd9.haoe5hsJyfHk1dQAHVK6N8dW_kfmtdbyuhlGwFdEHbM");
+
+    FCesiumCamera camera;
+    camera.ViewportSize = FVector2D(1024, 768);
+    camera.Location = FVector(0, 0, 0);
+    camera.Rotation = FRotator(-90.0, 0.0, 0.0);
+    camera.FieldOfViewDegrees = 90;
+    context.setCamera(camera);
+
+    context.georeference->SetGeoreferenceOriginLongitudeLatitudeHeight(
+        targetOrigin);
+
+    context.pawn->SetActorLocation(FVector(0, 0, 0));
+    context.pawn->SetActorRotation(FRotator(-90.0, 0.0, 0.0));
+
+    // Montreal Point Cloud
+    ACesium3DTileset* montrealTileset =
+        context.world->SpawnActor<ACesium3DTileset>();
+    montrealTileset->SetTilesetSource(ETilesetSource::FromCesiumIon);
+    montrealTileset->SetIonAssetID(28945);
+    montrealTileset->SetIonAccessToken(ionToken);
+    montrealTileset->SetMaximumScreenSpaceError(16.0);
+    montrealTileset->SetActorLabel(TEXT("Montreal Point Cloud"));
+
+    context.tilesets.push_back(montrealTileset);
+  };
+
+  auto after = [this](LoadTestContext& context) {
+    // Zoom way out
+    FCesiumCamera zoomedOut;
+    zoomedOut.ViewportSize = FVector2D(1024, 768);
+    zoomedOut.Location = FVector(0, 0, 7240000.0);
+    zoomedOut.Rotation = FRotator(-90.0, 0.0, 0.0);
+    zoomedOut.FieldOfViewDegrees = 90;
+    context.setCamera(zoomedOut);
+
+    context.pawn->SetActorLocation(zoomedOut.Location);
```

Comment:
It's weird that we need to `setCamera` and also set the DynamicPawn's location. Why do we need the custom camera at all?

Reply:
Good catch, we don't need it. Setting the pawn's location / rotation should be enough to run the test.

This was leftover setup code when I was playing with creating a world outside of the editor. It was needed this because `ACesium3DTileset` wasn't picking up any cameras in `::GetCameras()`.

Reply:
Actually, we do need it now. Just committed changes to start / end PIE mode, and in this mode, it does use this additional camera rather than the active viewport camera.